BeatSaberTwitchIntegration/BeatBot.cs

```diff
@@ -129,18 +130,15 @@ public void Initiate()
                                 var splitInput = inputLine.Split(' ');
                                 if (splitInput[0] == "PING")
                                 {
-                                    _logger.Info("Responded to twitch ping.");
                                     SendMessage("PONG " + splitInput[1]);
-                                    Thread.Sleep(1750);
+                                    continue;
```

Comment:
The Thread.Sleep(1750) is a lazy way to account for the twitch rate limiting. It should not be removed without an alternative being implemented.

Reply:
rate limit is with message queue solved